### ISSUE TITLE
Center the map on the user's location

### ISSUE THREAD

**Issue by nicoledaniels** [Pitt-CSC/club_project_2017#99]
Center the Leaflet map on the user's location if provided. The zoom should increase if the user's location is provided. If the location is not provided or geolocation services fails, set the center of the map to Cathy's coordinates.

**Comment by vonbearshark** [comment]
PR did not address some bugs, so I reverted.

There seems to be this weird bug on Firefox that if a user selects "Not Now", the error function is never called: https://bugzilla.mozilla.org/show_bug.cgi?id=675533.

Likewise, Chrome only supports geolocation through https, which we don't have on our custom domain (which is also a GH Pages site) right now: https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only. We'll need to set that up first-- @nicoledaniels do you have our Namecheap creds to look into that?

**Comment by brlodi** [comment]
^ That Bugzilla thread is a... well, several cursewords. I'm personally of the mind to say screw it for a bug that just got fixed for future release after *5.5 years* of explicitly refusing to implement part of a specification because the devs think Firefox is above it. That's the sort of crap that made me stop using Mozilla software.

**Comment by vonbearshark** [comment]
Yeah I'm totally cool with a Mozilla conditional that drops the prompt and uses Cathy for now.

**Comment by nicoledaniels** [comment]
Still working on getting the SSL certs for Google Chrome, but should be coming soon which will fix the problem
